tests/unit/lms/services/lti_outcomes_test.py

```diff
@@ -300,23 +239,12 @@ def requests(self, patch):
         return patch("lms.services.lti_outcomes.requests")
 
     @classmethod
-    def element_text(cls, xml, path):
-        element = LTIOutcomesClient.find_element(xml, path)
```

Comment:
Uses the code we are testing to run the tests. This seems like something to avoid.